setup.py

```diff
@@ -31,7 +31,8 @@
     ],
     include_package_data=True,
     install_requires=[
-        'Django>=1.10',
+        'Django==1.11;python_version<"3.5"',
+        'Django>=1.11,<2.2;python_version>"3.4"',
```

Comment:
@Eorhim I'm not sure about the python version restrictions specification matches the official Django one, can you please double check? https://docs.djangoproject.com/en/dev/faq/install/#what-python-version-can-i-use-with-django

based on that, I think it should be
* 1.11 => 2.7, 3.4-3.7
* 2.0 >= 3.4
* 2.1 >= 3.5

Reply:
I originally had that definition, but Django told me not to use Python 3.4:

`Django requires Python '>=3.5' but the running Python is 3.4.3`

Python 3.4 won't be supported after 2019-03-16 anyway.

Reply:
Wait. I had an error in the `tox.ini`. I forgot to reduce the version for Django 2.0. It actually was testing Django 2.1.